src/coap_tinydtls.c

```diff
@@ -27,9 +27,10 @@
 #undef PACKAGE_URL
 #undef PACKAGE_VERSION
 
-#include <tinydtls/tinydtls.h>
-#include <tinydtls/dtls.h>
-#include <tinydtls/dtls_debug.h>
+#include <tinydtls.h>
```

Comment:
IIRC we have deliberately changed the include path to have the prefix "tinydtls". I think removing it now will introduce a bunch of new errors.

Reply:
As it happens, it does not affect the posix, lwip and riot builds.  I have now special cased this for RIOT.
The long term right answer probably is for tinydtls have its own include/tinydtls/ directory for header files in the source tree.